src/config/mod.rs

```diff
@@ -414,6 +414,10 @@ pub struct WindowConfig {
     #[serde(default, deserialize_with = "failure_default")]
     dimensions: Dimensions,
 
+    /// Initial position
+    #[serde(default, deserialize_with = "failure_default")]
+    position: Option<Position>,
+
```

Comment:
Having the `dimensions` in the global `Config` is actually deprecated, I'm not sure why it hasn't been moved to the bottom and marked as such, but it will print a warning when it's used.

So the position should not be added here. I'd also appreciate it if you could move the `dimensions` to the bottom where the other items are and mark it with a deprecation comment, so this issue will not occur again.

Reply:
Should I add a changelog entry to document this? Or is the already-present `print_deprecation_warnings` enough?

Reply:
This is just a code change that does not affect the user. The warning is already present. So there's no need for a change log entry.